composer.json

```diff
@@ -16,27 +15,28 @@
             "homepage": "http://github.com/KnpLabs/KnpTimeBundle/contributors"
         }
     ],
-
     "require": {
-        "php":                      ">=5.3.3",
-        "symfony/framework-bundle": "~2.0"
+        "php": ">=5.3.3",
+        "symfony/dependency-injection": "^2.7",
```

Comment:
Good catch. You are right about this. Normally I take it as good practice to use the latest stable versions of the symfony components but here there seem to not be any breaking changes.

I have reverted the version constraints.